sample-queries/sample-queries.json

```diff
@@ -1390,36 +1390,40 @@
         "id": "7d3116a3-1690-4a15-ba47-153cb7891d00",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all conditional access policies",
+        "humanName": "List Conditional Access policies",
         "requestUrl": "/v1.0/identity/conditionalAccess/policies",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-policies?view=graph-rest-1.0&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "4fc821e4-f291-4944-999a-6d5b84ec8735",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all named locations",
+        "humanName": "List Named Locations",
         "requestUrl": "/v1.0/identity/conditionalAccess/namedLocations",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-namedlocations?view=graph-rest-1.0&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "473bd812-0df6-4fcb-bd23-d4b81eaa5f3e",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all conditional access policies (beta)",
+        "humanName": "List Conditional Access policies (beta)",
         "requestUrl": "/beta/identity/conditionalAccess/policies",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/conditionalaccessroot-list-policies?view=graph-rest-beta&tabs=http",
+        "tip": "We’d like to hear from you. Please leave your feedback on Conditional Access API here: https://aka.ms/caapifeedback",
         "skipTest": false
     },
     {
         "id": "d565b5f3-a268-4acf-a068-565294b5cdae",
         "category": "Security",
         "method": "GET",
-        "humanName": "get all named locations (beta)",
+        "humanName": "List Named Locations (beta)",
```

Comment:
human name should start with a small letter.

Reply:
@bettirosengugi  I have made the changes.